ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/config/Cli.scala

```diff
@@ -232,12 +251,18 @@ object Cli {
         } yield WorkflowConfig.StreamConfig.ActiveContractsStreamConfig(
           name = name,
           filters = filters,
-          objectives = Some(
-            WorkflowConfig.StreamConfig.RateObjectives(
-              minItemRate = minItemRate,
-              maxItemRate = maxItemRate,
-            )
-          ),
+          objectives = {
+            (minItemRate, maxItemRate) match {
```

Comment:
As above this wants a helper function to stay DRY

Reply:
5722bfb86199f66d90fd5c6d9bcdcb839c543e43